drivers/platform/stm32/stm32_gpio.c

```diff
@@ -42,6 +42,10 @@
 #include "no_os_gpio.h"
 #include "stm32_gpio.h"
 
+#ifndef GPIO_MODE && defined(STM32L5)
+#define GPIO_MODE (0x00000003U)
```

Comment:
nit: do we need brackets here?

Reply:
I left the brackets there because I copied the definition directly from STM HAL.

Reply:
I guess it depends on the family: (see line 142) https://www.disca.upv.es/aperles/arm_cortex_m3/llibre/st/STM32F439xx_User_Manual/stm32f4xx__hal__gpio_8c_source.html

Reply:
I removed the brackets now. Let's wait for @buha 's approval before merging.